assets/js/components/activation/activation-app.js

```diff
@@ -19,56 +19,20 @@
 /**
  * WordPress dependencies
  */
-import { Component, Fragment } from '@wordpress/element';
-
-/**
- * External dependencies
- */
-import { trackEvent } from 'GoogleUtil';
+import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
+// eslint-disable-next-line @wordpress/dependency-group
```

Comment:
Is this eslint exception still necessary?

Reply:
Yeah, the dependency stuff is a mess with the aliases we have. We really ought to sort out project-relative imports and remove all of them at some point…

Reply:
Agreed. I'll open an issue for that.

Reply:
See #1135